src/Markdown/Listener/CacheMarkdownListener.php

```diff
@@ -146,15 +146,21 @@ private function getMissingMentionsFromMarkdown(string $markdown): array
     /** @return string[] */
     private function getMissingMagazineMentions(string $markdown): array
     {
-        $words = preg_split('/[ \n\[\]()]/', $markdown);
+        $words = preg_split('/[ \n\[\]()]/', str_replace(chr(194).chr(160), "&nbsp;", $markdown));
         $missingCommunityMentions = [];
         foreach ($words as $word) {
             $matches = null;
-            if (preg_match('/'.CommunityLinkParser::COMMUNITY_REGEX.'/', $word, $matches)) {
+            $word2 = preg_replace('/[[:cntrl:]]/', '', $word);
+            if (preg_match('/'.CommunityLinkParser::COMMUNITY_REGEX.'/', $word2, $matches)) {
                 $apId = "$matches[1]@$matches[2]";
-                $magazine = $this->magazineRepository->findOneBy(['apId' => $apId]);
-                if (!$magazine) {
-                    $missingCommunityMentions[] = $apId;
+                $this->logger->debug("searching for magazine '{m}', original word: '{w}', word without cntrl: '{w2}'", ['m' => $apId, 'w' => $word, 'w2' => $word2]);
```

Comment:
cntrl?

Reply:
owh I see, you mean the `preg_replace` above. Like new line, tab, etc. Maybe add a comment